Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 4.0'
+gem 'rails', '~> 5.0.0'
+gem 'rails-controller-testing'
```

Comment:
> We should try to minimize dependencies wherever possible.

I agree. But, this gem is very useful for passing controller spec on both Rails4.x and Rails5. Besides, this gem is maintained by rails organization. I think that it is better to use it.